### ISSUE TITLE
Examples of operations for points, lines and polygons

### ISSUE THREAD

**Issue by alex-s-gardner** [JuliaGeo/GeometryOps.jl#219]
understanding geometry relations can be bit un-intuitive as we tend to use language like `overlaps` , `touches`, `within` more loosely in common language than the precise definitions for geometry operations. For that reason it would be helpful if examples for each relation/operation could be provided for points, lines and polygons to help build intuition.

**Comment by asinghvi17** [comment]
Agreed.  Should this go to the geocompjl book and we link / copy it?

**Comment by rafaqz** [comment]
This docs duplication is going to be an issue at some point... would be cool if we could make it all modular somehow

**Comment by alex-s-gardner** [comment]
Given the importance of these concepts to GeometryOps I think examples should live in "https://juliageo.org/GeometryOps.jl/stable/source/methods/geom_relations/ ..." Simply expanding on the examples already provided would go a long way. This would show the concepts and how to use the tools within juliageo... any future contributions to a book or common reference source would be made easier.

**Comment by rafaqz** [comment]
Yeah, I think we need them here too. If you have time to write them...
